CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ We use the following categories for changes:
 ### Fixed
 - Fix broken cache eviction in clockcache [#1603]
 - Possible goroutine leak due to unbuffered channel in select block [#1604]
+- `0 samples/sec` in throughput by logging metric metadata throughput [#1643]
```

Comment:
Should this be added as a change instead? We are changing from separate log lines for each telemetry type to a unified throughput log message.

Reply:
Yeah, it will be better as a change. Thank you.